spec/system/drawings/archive_spec.rb

```diff
@@ -3,9 +3,9 @@
 require "rails_helper"
 
 RSpec.feature "Drawings index page", type: :system do
-  fixtures :sites
-
-  let!(:site) { sites(:elm_grove) }
+  let!(:site) { create :site, address_1: "Elm Grove" }
+  let!(:assessor) { create :user, :assessor }
+  let!(:reviewer) { create :user, :reviewer }
```

Comment:
This is the tiniest comment in the world, but this is a good use-case of using `let` instead of `let!` for `assessor` and `reviewer`. That way, the specs that don't use the reviewer, won't create it in the database.

Of course in this situation, that will save us about 0.3 milliseconds (complete guess) from our test suite time, so it's really not a big deal, but I thought I'd mention it anyway.

Reply:
yes I wasn't sure what to go for.. since I've seen let! used in many files then let in others, I kinda just tried to replace everything that was let! with let! and so on